app/main/forms/frameworks.py

```diff
@@ -0,0 +1,31 @@
+from flask.ext.wtf import Form
+from wtforms import validators, StringField
+from wtforms import BooleanField as _BooleanField
+
+
+class BooleanField(_BooleanField):
+    """Tri-state boolean field
```

Comment:
I thought we were migrating from Grails to get away from things like tri-state booleans!